### PR TITLE
Release Note v10.7.0

### PR DESCRIPTION
**Pull request by GiovannaMonti**
## Pull Request Type

- [ ] Bugfix
- [ ] Documentation content changes
- [x] Release note
## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensible content has been committed

### CODE REVIEW

File: docs/release-notes/platform_10-7-0_releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+### Bug Fix
+
+## Fast Data
+
+### New Features
+
+#### Debezium CDC integrated in Fast Data
+
+From now on, thanks to the integration of the Debezium CDC in our Marketplace, some pre-packed plugins and templates can be used along with Fast Data with just few clicks! You can find a pre-configured plugin to be used with MySQL Databases and a pre-configured template to be used with Oracle Databases.
```
Comment (TheRusCompass) [inline docs/release-notes/platform_10-7-0_releasenotes.md:41]: @danibix95 pls have a look and feel free to add whatever you think is needed

File: docs/release-notes/platform_10-7-0_releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+#### Single View Creators
+
+The _Single View_ page has been updated with a new look for the Single View Creator section. This update features a clearer list of the microservices associated with the Single View, as well as a dedicated page for updating their configurations.
+
+### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+* A bug on the No Code configuration caused the Undo/Redo button to not be available when deleting a projection
+* The condition panel in the No Code configuration is now scrollable
```
Comment (ianFar96) [inline docs/release-notes/platform_10-7-0_releasenotes.md:59]: @TheRusCompass this goes here or in the console section?

File: docs/release-notes/platform_10-7-0_releasenotes.md (hunk 1 above)
Comment (ianFar96) [reply docs/release-notes/platform_10-7-0_releasenotes.md:43]: Should this go in to the Marketplace section?

Comment (danibix95) [reply docs/release-notes/platform_10-7-0_releasenotes.md]: LGTM 👍🏻

Should we add that the Real-Time Updater can be configured to accept Debezium change events (Fast Data ingestion messages in the format of Debezium)?

File: docs/release-notes/platform_10-7-0_releasenotes.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+* In the ingestion storer (v1.3.2) the post-ingestion topics are optional when when the forwarding of ingestion messages is disabled
```
Comment (TheRusCompass) [inline docs/release-notes/platform_10-7-0_releasenotes.md:68]:
```suggestion
* In the ingestion storer (v1.3.2) the post-ingestion topics are optional when the forwarding of ingestion messages is disabled
```

File: docs/release-notes/platform_10-7-0_releasenotes.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+Service Accounts can only be created by a Company Owner inside the Users portal section, which has now been renamed in "Identities".
+
+![Identities Table](./img/10.7/identities_table.png)
+
+Just like human Users, Service Accounts can be assigned Roles on Company, Project or Runtime Environment levels, based on which they will be able to perform different types of action.  
+
+To discover more about how to manage them, just take a look at the [Manage Service Accounts](/development_suite/identity-and-access-management/service-account-management) documentation page.
+
+#### Subscribing to console events
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-7-0_releasenotes.md:33]:
```suggestion
#### Subscription to Console Events via HTTP webhooks
```

File: docs/release-notes/platform_10-7-0_releasenotes.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+_March 30, 2023_
+
+## Console
+
+### New Features
+
+#### Machine-to-Machine Authentication
+
+From now on, thanks to Mia-Platform Console you can create and manage Service Account Users, a new type of identity in addition to traditional Users!
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-7-0_releasenotes.md:16]:
```suggestion
From now on, Mia-Platform Console supports Service Account Users, a new type of identity in addition to traditional Users!
```

File: docs/release-notes/platform_10-7-0_releasenotes.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+Components which perform http-calls (eg, `bk-crud-client`) have property `credentials`, which specifies credentials of http calls
+
+### Bug Fix
+
+#### Files are downloaded with correct name
+Files are downloaded with correct name, instead of applying "_" at the beginning and end of it.
+
+## How to update your Console
+
+For on-premise Console installations, please contact your Mia Platform referent to know how to use the `Helm chart version ?.?.?`.
```
Comment (albertotessarotto) [inline docs/release-notes/platform_10-7-0_releasenotes.md:102]: Remind to insert the correct version

File: docs/release-notes/platform_10-7-0_releasenotes.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+From today it will be possible to subscribe to some events on Mia-Platform Console via HTTP webhooks.  
+With this feature it will be possible to configure custom HTTP calls that will be triggered when the respective event takes place in Console.
+
+The supported events are:
+
+- Project Creation
+- Service Creation
+
+[Go to the documentation](/development_suite/..) page for more information.
```
Comment (marcofilippi) [inline docs/release-notes/platform_10-7-0_releasenotes.md:43]: to be updated with the correct link

**Review by FilippoRezzonico — APPROVED** [review]

**Pull request merged**